eng/pipelines/runtime.yml

```diff
@@ -302,6 +302,28 @@ jobs:
           eq(variables['monoContainsChange'], true),
           eq(variables['isFullMatrix'], true))
 
+# Build and test libraries under single-file publishing
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
+    buildConfig: Release
+    platforms:
+    - windows_x64
+    - Linux_x64
```

Comment:
It seems to be missing `osx_x64`, is it by design?

Reply:
There were some problems what I tested this earlier. Bringing Mac online would be good, but wasn't necessary for a first pass here.